yarn.lock

```diff
@@ -2386,6 +2365,30 @@
   dependencies:
     joi "^17.2.1"
 
+"@react-native-community/cli@12.3.2":
```

Comment:
It looks like we ended up with both `@react-native-community/cli` 12.x and 13.x here.

Reply:
Ah good catch, through react-native-xaml which still depends on @types/react-native, which brings in react-native * as a dep.  

https://github.com/microsoft/react-native-xaml/pull/279 will remove the now unnecessary @types/react-native dependency from react-native-xaml, then we can stop bringing in multiple react-native versions.